ephypype/nodes/import_data.py

```diff
@@ -448,3 +451,70 @@ def _list_outputs(self):
         outputs["fif_file"] = self.fif_file
 
         return outputs
+
+
+# ------------------- fif2npy --------------------------- #
+class Fif2TsInputSpec(BaseInterfaceInputSpec):
+    """Input specification for Fif2Ts."""
+
+    fif_file = File(exists=True, desc='fif file', mandatory=True)
+
+
+class Fif2TsOutputSpec(TraitedSpec):
+    """Output specification for Fif2Ts."""
+
+    ts_file = traits.File(exists=True, desc="time series in .npy format")
+    channel_coords_file = traits.File(
+        exists=True, desc="channels coordinates in .txt format")
+    channel_names_file = traits.File(
+        exists=True, desc="channels labels in .txt format")
+    sfreq = traits.Float(desc='sampling frequency', mandatory=True)
+
+
+class Fif2Ts(BaseInterface):
+    """Import Elekta raw data in .fif format.
+
+    Save the data time series in .npy format, as well as the channels names
+    and location in txt format.
+
+    Parameters
+    ----------
+    fif_file
+        type = File, exists=True, desc='fif file', mandatory=True
+
+    Returns
+    -------
+    ts_file
+        type  = File, exists=True, desc="data time series in .npy format"
+    channel_coords_file
+        type = File, exists=True, desc="channels coordinates in txt format"
+    channel_names_file
+        type = File, exists=True, desc="channels names in txt format"
+    sfreq
+        type = Float, desc="sampling frequency"
```

Comment:
How can class return parameters?

Reply:
I tested  [plot_connectivity](https://github.com/neuropycon/ephypype/blob/75f5eb03f985196e3da636ff7ae628f59c2a1f0e/examples/plot_connectivity.py) with the new merge #52 and it works. 

The proposed change
```
Parameters -> Interface Input
Returns -> Interface Output
```
can be done in the #56

Reply:
okay but at least use [numpy docstring conventions](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_numpy.html) and specify the type of the arguments in a standard way

Reply:
> okay but at least use [numpy docstring conventions](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_numpy.html) and specify the type of the arguments in a standard way

ok! I'll do it in #56 .
Is it ok also for you @davidmeunier79 ?